src/scales/scale.time.js

```diff
@@ -426,6 +430,9 @@ var defaultConfig = {
 	 */
 	bounds: 'data',
 
+	adapters: {
```

Comment:
Here, I would just define `adapters: {}` as a default placeholder then leave `undefined` the date adapter options (which will fallback to `{}` in the date adapter constructor).

Reply:
done